config/packages/tabler.yaml

```diff
@@ -44,35 +44,34 @@ tabler:
 
     icons:
         # used for the action_collapsebutton components
-        collapse: fas fa-chevron-down
+        collapse: "fa6-solid:chevron-down"
         # used for the password-reset template
-        mail: far fa-envelope
+        mail: "fa6-regular:envelope"
         # used for the 404/500/... error page
-        back: fas fa-long-arrow-alt-left
+        back: "fa6-solid:left-long"
```

Comment:
Do we still keep the `icons` config part in the future, or just during the temporary "deprecated" period?

Reply:
Keep it during the deprecation period. I want to try to make this PR completely BC safe.

Reply:
On the other hand, this is a tiny layer on top of the UX icons later, that helps managing and downloading them. 
Not sure yet ...